modules/tinymce/src/themes/silver/test/ts/browser/editor/core/UndoRedoUiTest.ts

```diff
@@ -0,0 +1,59 @@
+import { UiFinder, Waiter } from '@ephox/agar';
+import { describe, it } from '@ephox/bedrock-client';
+import { SugarBody } from '@ephox/sugar';
+import { TinyHooks, TinyUiActions } from '@ephox/wrap-mcagar';
+
+import Editor from 'tinymce/core/api/Editor';
+
+describe('browser.tinymce.themes.silver.editor.core.UndoRedoUiTest', () => {
```

Comment:
```suggestion
describe('browser.tinymce.themes.silver.editor.core.UndoRedoTest', () => {
```
Can you please also rename the file to match this, as we normally try to make it match the module name, which in this case is just `UndoRedo` (similar to `SimpleControls.ts` and `SimpleControlsTest.ts`). The other reason is that the UI part in the name is also largely redundant, as the whole theme is related to the UI.